### ISSUE TITLE
rgbgfx being the bane of my existence as usual

### ISSUE THREAD

**Issue by ZoomTen** [ZoomTen/a-lot-of-rgbds#1]
```
rgbgfx -c '#ffffff,#aaaaaa,#555555,#000000' -d 2 -Z -c embedded -o gfx/trainers/gold.2bpp -- gfx/trainers/gold.png
Assertion failed: options.palSpec.size() == 1 (src/gfx/process.cpp: generatePalSpec: 521)
make: *** [Makefile:241: gfx/trainers/gold.2bpp] Aborted (core dumped)
```
Preliminary guesses:
1. I compiled it wrong and forgot to update libpng/zlib.
2. Should've used glibc like upstream does.
3. I don't even know

**Comment by ZoomTen** [comment]
one solution; works with win32 at least (`src/gfx/process.cpp`, 0.7.0 line 521)
```diff
+	options.palSpec.clear();
 	options.palSpec.emplace_back(); // A single palette, with `#00000000`s (transparent)
```

**Comment by ZoomTen** [comment]
Resolved in gbdev/rgbds#1415
(Now I gotta backport this fix as a patch for usability reasons…)
